samples/snippets/type_system_test.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_type_system_examples() -> None:
+    # [START bigquery_dataframes_type_sytem_local_type_conversion]
```

Comment:
Region tags need to be globally unique. Maybe it makes more sense to include "timestamp" somewhere in here? I imagine `bigquery_dataframes_type_sytem_local_type_conversion` could apply to lots of different types beyond the scope of this sample.

Reply:
Right. Changed it to "..._timestamp_local_type_conversion"